frontend/src/metabase/visualizations/components/TableInteractive/TableInteractive.jsx

```diff
@@ -1045,6 +1077,7 @@ class TableInteractive extends Component {
             }
             return (
               <TableInteractiveRoot
+                bg={backgroundColor || "white"}
```

Comment:
@WiNloSt I see what you mean! Currently we only pass the `theme.other` defaults in embedding environments, but not in normal webapp environment - that is why I added a "white" fallback that would get apply to the dashboard. 

My mental model is that `theme.other` would never exist in normal webapp environment, so the values of `theme.other.*` would always only refer to embedding. I consider `cell.backgroundColor` as an override in this case.

I can see that if we also supply the same default values for `theme.other` to the webapp environment, then I think that would make it so that we don't need to supply the `|| "white"` defaults - so maybe that is a better solution in the long run.

PS. I just noticed the `|| "white"` fallback is duplicated in both places - removed one of them.

Reply:
My idea was that we use `theme.other` to override certain elements for SDK users, but leave the behavior of the dashboard including static embedding the same (point 1 in my previous comment). This allows us to be confident that we won't break static embedding usecases since we also have `theme=transparent` which currently doesn't look right, but customers could be using that and don't expect the behavior to change.

I'll keep reviewing whether this PR changes the dashboard behavior outside SDK embedding context, because if it does, we should communicate with product folks about this because there's a project [aiming to improve static embed theme](https://www.notion.so/metabase/Re-design-transparent-theme-for-static-embedding-a28e98a6e5c64301a48387c7b333d569). So in the meantime, it'd be safe to ensure we don't accidentally change the dashboard behavior for static embeds.

Reply:
I see! Would it make sense to you if I modify this PR to load theme.other into the dashboard environment, while making sure the default theme config is the same as the currently dashboard.

I think that would not affect both the dashboard and the embeds if the value are the same as today - and we can get rid of the fallbacks.

Reply:
I think it's safer that way. Let me confirm
1. we have another `theme.other` specifically for dashboard this applies to normal app dashboards, embed dashboards
2. we still have `theme.other` for SDK users which contains the better default values e.g. white background

if that's the case then let's do it.

Reply:
Yep, sounds good! Will update the PR today. Will keep the "transparent" theme settings in mind when I test.